mailpoet/RoboFile.php

```diff
@@ -311,6 +311,10 @@ public function testPerformance($path = null, $opts = ['url' => null, 'pw' => nu
       ->option('env', 'PW=' . $opts['pw'])
       ->option('env', 'HEADLESS=' . ($opts['head'] ? 'false' : 'true'))
       ->option('env', 'SCENARIO=' . $opts['scenario'])
+      ->option('env', 'K6_CLOUD_TOKEN=' . getenv('K6_CLOUD_TOKEN'))
+      ->option('env', 'K6_CLOUD_ID=' . getenv('K6_CLOUD_ID'))
+      ->option('env', 'K6_PROJECT_NAME=' . $opts['name'])
```

Comment:
Can we just use `$opts['scenario']` here and remove the `name` parameter?

Reply:
I just tried it locally and it won't work.. especially if we want to add some custom naming, like I mentioned below `--name=localjanjakes` for example.. Let's leave it then as-is so it's customizable. @JanJakes

Reply:
Having to use a custom `--name` locally would be very unintuitive. See also my comment https://github.com/mailpoet/mailpoet/pull/4835#discussion_r1160590017. I still think we should remove this option to keep things as simple as possible.

Reply:
Sure, removed it 👍